src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -19,21 +19,15 @@ def write_caption_data_to_file(caption_data: dict, file_name: str):
        :param caption_data: json with byte pairs
        :param file_name: used for saving resulting file
     """
-    # datetime object containing current date and time
-    now = datetime.now()
-    # mm.dd.YY_H:M:S
-    dt_string = now.strftime("%m.%d.%Y_%H-%M-%S")
-
     path = config.path_to_data_folder
-    file_name = file_name + f'output_{dt_string}.json'
     try:
         with open(path + file_name, 'w', encoding='utf-8') as file:
             json.dump(caption_data, file, ensure_ascii=False, indent=4)
     except IOError as err:
         logging.error(f'Could not write JSON to file: {err}')
 
 
-def consume(caption_data: dict):
+def consume(caption_data: dict, time_stamp: str):
```

Comment:
Consume is also called by main. you need to change it there as well or main will no longer work.